README.md

```diff
@@ -16,10 +22,15 @@
 
 #### To run a dev server:
 
-1. Start ganache
-    * Either using the app
-    * Or by running `ganache-cli` in a different tab
-2. Run the server:
+1. Start ganache by running
+    ```
+    ganache-cli
+    ```
+2. **In a new tab** Run the server
+    ```
+    yarn start
+    ```
+    After seeing the message "compiled successfully!", kill the server <kbd>cmd</kbd>+<kbd>C</kbd> and run it again
```

Comment:
@geoknee do you know why this step is needed?

Reply:
There is an open issue with a partial explanation.